clusterloader2/pkg/prometheus/prometheus.go

```diff
@@ -154,6 +156,12 @@ func NewController(clusterLoaderConfig *config.ClusterLoaderConfig) (pc *Control
 		// Backward compatibility.
 		clusterLoaderConfig.PrometheusConfig.ScrapeNodeExporter = mapping["PROMETHEUS_SCRAPE_NODE_EXPORTER"].(bool)
 	}
+	if _, exists := mapping["PROMETHEUS_SCRAPE_WINDOWS_NODE_EXPORTER"]; !exists {
+		mapping["PROMETHEUS_SCRAPE_WINDOWS_NODE_EXPORTER"] = clusterLoaderConfig.PrometheusConfig.ScrapeWindowsNodeExporter
+	} else {
+		// Backward compatibility.
+		clusterLoaderConfig.PrometheusConfig.ScrapeWindowsNodeExporter = mapping["PROMETHEUS_SCRAPE_WINDOWS_NODE_EXPORTER"].(bool)
```

Comment:
If you are keeping this for backwards compat I think you would also need to keep `additionalScrapeConfigs` that set the windows node ip in the file [clusterloader2/pkg/prometheus/manifests/prometheus-prometheus.yaml](https://github.com/kubernetes/perf-tests/pull/2041/files#diff-50b6a49c70a72aec86e5668edf14744f9ea763b5fd8fdfa2eeb62d131529ceee)?

Reply:
I added them back